### DISCUSSION TITLE
Mail.sendLater doesn't show up in IDE IntelliSense

### DISCUSSION THREAD

**Discussion by msrumon** [adonisjs/core#1274] [General]
Following [this doc](https://preview.adonisjs.com/guides/mail#defer-email-sending) to queue emails in-memory for sending, but my IDE is not showing the required `sendLater` method:

![image](https://user-images.githubusercontent.com/9024015/87914414-6793d900-ca92-11ea-85fa-8079b57f0c1c.png)

The method is defined in `MailerContract` interface:
https://github.com/adonisjs/mail/blob/45f699319359fe880841a1ca357691ea7ac50d71/adonis-typings/mail.ts#L576-L579
...but not in `MailManagerContract` (because when I hover on `import Mail from '@ioc:Adonis/Addons/Mail'`, it shows its type `MailManagerContract`).

**Comment by thetutlage** [comment]
Mind sharing a PR for same?

**Reply by msrumon** [reply]
Where will I make the PR? This repo or `adonisjs/mail`?

**Reply by thetutlage** [reply]
The https://github.com/adonisjs/mail repo. The develop branch

**Reply by msrumon** [reply]
https://github.com/adonisjs/mail/pull/53

**Reply by msrumon** [reply]
I'm not very good at writing type definitions. Please explain to me if I did any mistake. I will be more than happy to learn. ;)
